package.json

```diff
@@ -324,14 +324,14 @@
     "@metamask/rpc-errors": "^7.0.0",
     "@metamask/safe-event-emitter": "^3.1.1",
     "@metamask/scure-bip39": "^2.0.3",
-    "@metamask/selected-network-controller": "^19.0.0",
+    "@metamask/selected-network-controller": "^22.1.0",
```

Comment:
Do we need to handle any of these breaking changes?

Reply:
Doesn't seem like it. 20.0.0 has breaking changes which were reverted in 21.0.0, and the other breaking changes are bumps to `@metamask/network-controller`.